app/jobs/send_article_endorsement_job.rb

```diff
@@ -1,9 +1,10 @@
-class SendArticleEndorsementJob < Struct.new(:endorsement_id)
-  def perform
-    endorsement = ArticleEndorsement.find(endorsement_id)
-    contributors = endorsement.article.contributors
-    article = endorsement.article
-    endorser = endorsement.user
+class SendArticleEndorsementJob < ActiveJob::Base
+  queue_as :default
```

Comment:
I'll miss the documentation of expected parameters in the class definition so can you add a comment with required arguments?